docs/_data/footer.yml

```diff
@@ -15,8 +15,8 @@ top:
 # Configuration for "Edit this page" link in the footer.
 # Customize with 'text' property.
 # Uncomment the following lines to show.
-#edit_page:
-#  text: Edit this page
+edit_page:
+  text: Edit this page
```

Comment:
```suggestion
edit_page:
  text: Edit this page on GitHub
```

Reply:
if we're going to be cool kids, we should call it out completely =P

Reply:
fixed